src/lib/data/dbUsers.ts

```diff
@@ -8,10 +8,9 @@ export async function dbInsertUser(
   const sql = `
   INSERT INTO users (
     user_hashed_email,
-    user_encrypted_pii,
-    user_creation_time
+    user_encrypted_pii
   )
-  VALUES ($1, $2, CURRENT_TIMESTAMP)
+  VALUES ($1, $2)
```

Comment:
Perhaps have one test to check that the creation time is set to an expected value.

Reply:
We already have a Truthy assertion.

For example...
```
        expect(user.userCreationTime).toBeTruthy();
```